pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -95,9 +95,28 @@ in mkDerivation {
         cp $icon $out/share/icons/hicolor/$path/apps/us.zoom.Zoom.png
     done
 
-    ln -s $out/share/zoom-us/zoom $out/bin/zoom-us
+
+    # Do the manual Qt wrapping, because existing hook doesn't support our usecase
+    qtWrapperArgs+=(--prefix XDG_DATA_DIRS : "$out/share/zoom-us")
```

Comment:
can use use the nix attr below for this, or does it not work? (I believe you can use placeholder)

Reply:
After inspecting the produced wrappers they're missing an entry to XDG_DATA_DIRS for `$out/share` which **needs** to be in these wrappers too. I see code in wrapQtAppsHook for that, so this confuses me.

Can you also add a comment why we need the `zoom-us` subdir in XDG_DATA_DIRS?

Reply:
I also believe you're wrapping in the wrong phase, can this be done in `preFixup`?

Reply:
@worldofpeace 
> I also believe you're wrapping in the wrong phase, can this be done in preFixup?

changed to `postFixup`, as I see there are some hooks added in `preFixup` in wrap-qt-setup-hook.sh. I don't know what will be the run order, but postFixup will definitely run after.